R/create.R

```diff
@@ -33,6 +33,12 @@
 #' src <- src_mongo()
 #' docdb_create(src, key = "mtcars", value = mtcars)
 #' docdb_get(src, "mtcars")
+#' 
+#' # SQLite
+#' src <- src_sqlite()
+#' docdb_create(src, key = "mtcars", value = mtcars)
+#' docdb_create(src, key = "mtcars", value = contacts)
```

Comment:
Currently, the package is setup to work with data.frame's - so e.g., the example with the contacts dataset fails

I think we'd prefer to keep the pkg as simple as possible and not try to support other data types

Reply:
Many thanks @sckott for review and suggestions! 
Ok - I have changed the proposal to work exclusively with data.frames. 
When a data.frame contains JSON strings, these are specially processed.